gazelle/def.bzl

```diff
@@ -15,5 +15,13 @@
 """This module contains the Gazelle runtime dependencies for the Python extension.
 """
 
+load("@bazel_skylib//lib:modules.bzl", "modules")
+load(":deps.bzl", "python_stdlib_list_deps")
+
 GAZELLE_PYTHON_RUNTIME_DEPS = [
 ]
+
+non_module_deps = modules.as_extension(
```

Comment:
Could you please move this into `gazelle/python/private:extensions.bzl`? please? The `modules.as_extension` is something that is not our external API within the gazelle plugin, so it should not be a part of `def.bzl`.